src/core/a-entity.js

```diff
@@ -121,12 +121,11 @@ var proto = Object.create(ANode.prototype, {
       var diff = oldMixinsIds.filter(function (i) { return newMixinsIds.indexOf(i) < 0; });
       // Remove the mixins that are gone on update
       diff.forEach(function (mixinId) {
-        var forEach = Array.prototype.forEach;
-        // State Mixins
-        var stateMixinsEls = document.querySelectorAll('[id^=' + mixinId + '-]');
-        var stateMixinIds = [];
-        forEach.call(stateMixinsEls, function (el) { stateMixinIds.push(el.id); });
-        stateMixinIds.forEach(self.unregisterMixin.bind(self));
+        // State mixins
+        var stateMixinsEls = Array.from(document.querySelectorAll('[id^=' + mixinId + '-]'));
```

Comment:
It seems that the support of `Array.from` is more limited than the previous form.